src/trail.js

```diff
@@ -0,0 +1,237 @@
+import {path} from "d3-path";
+import constant from "./constant.js";
+import {x as pointX, y as pointY, z as pointSize} from "./point.js";
+import {abs, atan2, cos, pi, tau, sin, sqrt, max, min} from "./math.js";
+
+export default function(x, y, size){
+    var defined = constant(true),
+        context = null,
+        ready, 
+        ready2,
+        scaleRatio_min = 1,
+        ct1,
+        px1, py1, pr1;
+    
+    x = typeof x === "function" ? x : (x === undefined) ? pointX : constant(x);
+    y = typeof y === "function" ? y : (y === undefined) ? pointY : constant(y);
+    size = typeof size === "function" ? size : (size === undefined) ? pointSize : constant(size);
+    
+    function crossProduct(x1, y1, x2, y2){
+        return x1*y2 - x2*y1;
+    }
```

Comment:
```suggestion
    function crossProduct(x1, y1, x2, y2) {
        return x1 * y2 - x2 * y1;
    }
```

another style change: spaces

Reply:
also elsewhere in D3's code base we use "cross" and "dot" rather than "crossProduct"